BaseFilter/sections/antiadblock.txt

```diff
@@ -1816,15 +1816,15 @@ cutt.net#%#//scriptlet('prevent-fetch', 'pagead2.googlesyndication.com')
 @@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=cutt.net
 ! https://www.reddit.com/r/uBlockOrigin/comments/xmjm98/zshort_detecting_adblocker_not_allowing_downloads/
 ! https://github.com/AdguardTeam/AdguardFilters/issues/115595
-charexempire.com,up-load.one#%#//scriptlet("abort-current-inline-script", "document.addEventListener", "/abisuq/")
+prz.pw,charexempire.com,up-load.one#%#//scriptlet("abort-current-inline-script", "document.addEventListener", "/abisuq/")
```

Comment:
Actually, it's `antiblock.org g207`.
You can find the rules using search `! SECTION: antiblock.org g207`.
What you need to do:
1. Remove your rules (just the domain)
2. Add the domain to the current scriptlet `#%#//scriptlet('abort-current-inline-script', 'document.addEventListener', 'displayMessage:')`
3. Add `@@||prz.pw^$generichide` to the content_blocker.txt. You'll find the same category there.